### PR TITLE
Cleaning up examples

### PR DESCRIPTION
**Pull request by Temikus**
Take 1 on #31, some are still left as they are
- Made the examples easily accessible.
- Removed old and redundant ones.
- Updated resources, images, etc.

/CC @plribeiro3000

### CODE REVIEW

File: examples/eric-fail.rb [hunk 1]
```diff
@@ -1,32 +0,0 @@
-def test
-  connection = Fog::Compute.new(:provider => "Google")
```
Comment (icco) [inline examples/eric-fail.rb]: lol, I guess we can stop making fun of @erjohnso in the fog-google examples :p

**Comment by erjohnso** [comment]
@icco - there is no legitimate reason I can think of to stop making fun of me.

File: examples/eric-fail.rb (hunk 1 above)
Comment (plribeiro3000) [inline examples/eric-fail.rb]: lol. did not know about this file at all. :sweat_smile:

Comment (icco) [inline examples/eric-fail.rb]: I can't remember the original bug, but there was some weird way that @erjohnso was using Fog in another library, which caused this script to blow up. It was interesting at the time, now it's just funny :p

**Pull request merged**